adminguide.rst

```diff
@@ -320,6 +322,52 @@ job queue and record of past jobs.
     If this happens it is best to stop Flux, remove the content.sqlite file,
     and restart.
 
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+flux-accounting configuration
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In order to support fairshare and multi-factor priority ordering of user jobs, the flux-accounting package must be present and configured for the system instance. This allows for the management of users and banks on a cluster, job priorities to be calculated using multiple factors and uses job record information from users to calculate and maintain fairshare values.
+
+The first thing to configure when first setting up the flux-accounting database is to set the PriorityUsageResetPeriod and PriorityDecayHalfLife parameters. Both of these parameters represent a number of weeks by which to hold usage factors up to the time period where jobs no longer play a factor in calculating a usage factor. If these parameters are not passed when creating the DB, PriorityDecayHalfLife is set to 1 week and PriorityUsageResetPeriod is set to 4 weeks, i.e the flux-accounting database will store up to a month's worth of jobs broken up into one week chunks:
+
+.. code-block:: console
+
+ $ flux account create-db --priority-decay-half-life=2 --priority-usage-reset-period=8 path/to/DB
+
+The other component to load is the multi-factor priority plugin, which can be loaded with ``flux jobtap load``:
+
+.. code-block:: console
+
+ $ flux jobtap load mf_priority.so
+
+This plugin can also be loaded by specifying it in the ``job-manager.plugins`` section of the Flux TOML configuration file. Example:
```

Comment:
This is how all system instance configurations will want configure the priority plugin, so it might be appropriate to drop the sentence above that shows how to load it manually.